test/traveler-test.js

```diff
@@ -1,13 +1,12 @@
 import chai from 'chai';
 const expect = chai.expect;
 
-import Traveler from '../src/users/traveler'
+import Traveler from '../src/jsClasses/traveler'
 
 describe('See if the tests are running', function() {
```

Comment:
totally see what you were doing! just don't forget you might want to change the describe statement later